src/native/managed/cdacreader/src/Legacy/ISOSDacInterface.cs

```diff
@@ -556,19 +556,12 @@ internal unsafe partial interface ISOSDacInterface12
     int GetGlobalAllocationContext(ulong* allocPtr, ulong* allocLimit);
 }
 
-internal struct VISITHEAP
-{
-    public ulong blockData;
-    public nuint blockSize;
-    public Interop.BOOL blockIsCurrentBlock;
-}
-
 [GeneratedComInterface]
 [Guid("3176a8ed-597b-4f54-a71f-83695c6a8c5e")]
 internal unsafe partial interface ISOSDacInterface13
 {
     [PreserveSig]
-    int TraverseLoaderHeap(ulong loaderHeapAddr, /*LoaderHeapKind*/ int kind, VISITHEAP pCallback);
+    int TraverseLoaderHeap(ulong loaderHeapAddr, /*LoaderHeapKind*/ int kind, /*VISITHEAP*/ delegate* unmanaged<ulong, nuint, Interop.BOOL> pCallback);
```

Comment:
So the `nuint` here is `size_t` in C++. This means if the cdac is reading a 32-bit DMP, it will be wrong if the reader process is 64-bit. Have we thought about this issue? Entirely possible this is a known limitation of the cdac.

Reply:
Okay.